cmd/podman/varlink/io.podman.varlink

```diff
@@ -59,6 +59,7 @@ type VolumeRemoveOpts (
 
 type Image (
   id: string,
+  digest:   string,
```

Comment:
when you change this file, you need to run a make varlink_api_generate to update the API.md documentation and include it in your commit.